xml/System.Formats.Tar/TarWriter.xml

```diff
@@ -208,36 +200,37 @@ The recommended format is <xref:System.Formats.Tar.TarEntryFormat.Pax> for its f
         <param name="entry">The tar entry to write.</param>
         <summary>Writes the specified entry into the archive stream.</summary>
         <remarks>
-          <format type="text/markdown"><![CDATA[
-
-## Remarks
-
-Before writing an entry to the archive, if you wrote data into the entry's <xref:System.Formats.Tar.TarEntry.DataStream>, make sure to rewind it to the desired start position.
-
-These are the entry types supported for writing on each format:
-
-- <xref:System.Formats.Tar.TarEntryFormat.V7>
-
-   - <xref:System.Formats.Tar.TarEntryType.Directory>
-   - <xref:System.Formats.Tar.TarEntryType.HardLink>
-   - <xref:System.Formats.Tar.TarEntryType.SymbolicLink>
-   - <xref:System.Formats.Tar.TarEntryType.V7RegularFile>
-
-- <xref:System.Formats.Tar.TarEntryFormat.Ustar>, <xref:System.Formats.Tar.TarEntryFormat.Pax>, and <xref:System.Formats.Tar.TarEntryFormat.Gnu>
-
-   - <xref:System.Formats.Tar.TarEntryType.BlockDevice>
-   - <xref:System.Formats.Tar.TarEntryType.CharacterDevice>
-   - <xref:System.Formats.Tar.TarEntryType.Directory>
-   - <xref:System.Formats.Tar.TarEntryType.Fifo>
-   - <xref:System.Formats.Tar.TarEntryType.HardLink>
-   - <xref:System.Formats.Tar.TarEntryType.RegularFile>
-   - <xref:System.Formats.Tar.TarEntryType.SymbolicLink>
-
-          ]]></format>
+          <para>Before writing an entry to the archive, if you wrote data into the entry's <see cref="P:System.Formats.Tar.TarEntry.DataStream" />, make sure to rewind it to the desired start position.</para>
+          <para>These are the entry types supported for writing on each format:</para>
+          <list type="bullet">
+            <item>
+              <para><see cref="F:System.Formats.Tar.TarEntryFormat.V7" /></para>
```

Comment:
Would be good to check this formatting in the preview.

Reply:
🤞

Reply:
Didn't work. Bummer. I'll fix this tomorrow.

![Screenshot_20220912-214300_Edge](https://user-images.githubusercontent.com/1175054/189810719-13959cda-74b5-49e5-b68a-f8435372e81b.jpg)

Reply:
You were right @gewarren, as usual. These tables with nested items should be ported as markdown. I'll make sure to special-case it in the docs porting tool.